package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "pie-monorepo",
   "description": "JustEatTakeaway",
-  "version": "1.22.0",
```

Comment:
We'll no longer be versioning the root of the monorepo, as it's not a package that is versioned / used by consumers.

Reply:
Just updating this – we'll be updating the root for root changes, but not through Changesets. It'll keep the manual workflow that we currently have.